events/patc.py

```diff
@@ -0,0 +1,160 @@
+import logging
+import posixpath
+import requests
+
+from bs4 import BeautifulSoup
+from dateutil.parser import parse
+import more_itertools as mi
+import pandas as pd
+import json 
+
+URL_BASE = "https://www.patc.net/PATC/Calendar/PATC/"
+CALENDAR_BASE_URL = ("Custom/Calendar.aspx?hkey" 
+                     "=9fc06544-1c54-4a47-9efc-8fcd2420a646")
+KEYWORDS = ["Start Time", "Start Date", "Event Category", "www"]
+RENAME_MAP = {
+    "all_day_event": "All Day Event",
+    "currency": "Event Currency Symbol",
+    "Description": "Event Description",
+    "end_date": "Event End Date",
+    "end_time": "Event End Time",
+    "event_cost": "Event Cost",
+    "event_organizers": "Event Organizers",
+    "start_date": "Event Start Date",
+    "start_time": "Event Start Time",
+    "time_zone": "Timezone",
+    "venue": "Event Venue Name",
+    "www": "Event Website",
+}
+
+
+def make_request_content(url_subpath):
+    """Make a request to a URL and return the Soup contents
+
+    :params str url_subpath:
+        String denoting subpath of the path to retrieve data from
+
+    :return:
+        A soup containing information about the page requested
+
+    :rtype:
+        bs4.BeautifulSoup
+    """
+    parse_link = posixpath.join(URL_BASE, url_subpath)
+    res = requests.get(parse_link)
+    res.raise_for_status()
+    soup = BeautifulSoup(res.content, features="lxml")
```

Comment:
Change this line to `soup = BeautifulSoup(res.content, 'html.parser')` since the lxml parser needs to be downloaded as an external dependency.